MacrobenchmarkSample/macrobenchmark/src/main/java/com/example/macrobenchmark/startup/SampleStartupBenchmark.kt

```diff
@@ -36,7 +37,12 @@ class SampleStartupBenchmark {
     fun startup() = benchmarkRule.measureRepeated(
         packageName = TARGET_PACKAGE,
         metrics = listOf(StartupTimingMetric()),
+        startupMode = StartupMode.COLD,
         iterations = 5,
+        setupBlock = {
```

Comment:
AFAICT with `COLD` startup s we don't need to `pressHome()` because the activity will be created fresh for each run.

Reply:
You're right 🤦🏼‍♂️  .. I'll remove it

Reply:
You don't need to put it in the setup block - typically we put it in the measureBlock (regardless of startup type) for consistency and brevity

Reply:
Actually, Rahul found it should be part of the `setupBlock`, otherwise it may cause some interference with the launcher app.  More [info]( https://github.com/android/compose-samples/pull/748/files/be0771898e29e161197c44f0d20263798940d412#r821146417).